### ISSUE TITLE
[1.4-beta.1] libsass binding was not found 

### ISSUE THREAD

**Issue by ramijarrar** [fourseven/meteor-scss#184]
Testing with the 1.4 beta, stuck on this error:

```
 While loading plugin `compileScssBatch` from package `fourseven:scss`:

   /Users/moe-talos/.meteor/packages/fourseven_scss/.3.4.3.nxxo7f++os.osx.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.osx.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/lib/extensions.js:158:11:
   The `libsass` binding was not found in
   /Users/moe-talos/.meteor/packages/fourseven_scss/.3.4.3.nxxo7f++os.osx.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.osx.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/vendor/darwin-x64-46/binding.node
   This usually happens because your node version has changed.
   Run `npm rebuild node-sass` to build the binding for your current node
   version.
   at Object.sass.getBinaryPath
   (/Users/moe-talos/.meteor/packages/fourseven_scss/.3.4.3.nxxo7f++os.osx.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.osx.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/lib/extensions.js:158:11)
   at Object.<anonymous>
   (/Users/moe-talos/.meteor/packages/fourseven_scss/.3.4.3.nxxo7f++os.osx.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.osx.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/lib/index.js:16:36)
   at Module._compile (module.js:409:26)
   at Object.Module._extensions..js (module.js:416:10)
   at Module.load (module.js:343:32)
   at Module.Mp.load
   (/Users/moe-talos/.meteor/packages/meteor-tool/.1.4.0-beta.1.1us55by++os.osx.x86_64+web.browser+web.cordova/mt-os.osx.x86_64/dev_bundle/lib/node_modules/reify/node/runtime.js:16:23)
   at Function.Module._load (module.js:300:12)
   at Module.require (module.js:353:17)
   at require (internal/module.js:12:17)
   at Object.<anonymous> (/tools/isobuild/bundler.js:1570:22)
   at Object.require (/tools/isobuild/bundler.js:1547:11)
   at meteorInstall.node_modules.meteor.compileScssBatch.plugin.compile-scss.js
   (packages/compileScssBatch/plugin/compile-scss.js:3:18)
   at fileEvaluate (packages/modules-runtime.js:158:9)
   at require (packages/modules-runtime.js:92:16)
   at <runJavaScript-29>:349:1
   at <runJavaScript-29>:355:3
```

**Comment by sebakerckhof** [comment]
I would suspect this is a bug with Meteor, which compiles native modules itself starting from 1.4.
I'll see if I can reproduce this.

**Comment by mikestokes** [comment]
Same with me on Meteor 1.4-beta 1 running OS X El Capitan.

**Comment by meteorza** [comment]
Same problem for me. I use poetic:materialize-scss which auto installs fourseven:scss, I get the same error using meteor 1.4 beta

**Comment by sebakerckhof** [comment]
@meteorza also running on mac?

**Comment by meteorza** [comment]
No. 

> uname -a
Linux ubuntu 4.4.0-21-generic #37-Ubuntu SMP Mon Apr 18 18:33:37 UTC 2016 x86_64 x86_64 x86_64 GNU/Linux

> lsb_release -a
No LSB modules are available.
Distributor ID: Ubuntu
Description:    Ubuntu 16.04 LTS
Release:        16.04
Codename:       xenial

**Comment by meteorza** [comment]
Sorry for short reply earlier, was on my way out. Here is detailed error message:

 ```
While loading plugin `compileScssBatch` from package `fourseven:scss`:

   /root/meteor/.meteor/packages/fourseven_scss/.3.4.3.x0c10u++os.linux.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.linux.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/lib/extensions.js:158:11:
   The `libsass` binding was not found in
   /root/meteor/.meteor/packages/fourseven_scss/.3.4.3.x0c10u++os.linux.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.linux.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/vendor/linux-x64-46/binding.node
   This usually happens because your node version has changed.
   Run `npm rebuild node-sass` to build the binding for your current node version.
   at Object.sass.getBinaryPath
   (/root/meteor/.meteor/packages/fourseven_scss/.3.4.3.x0c10u++os.linux.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.linux.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/lib/extensions.js:158:11)
   at Object.<anonymous>
   (/root/meteor/.meteor/packages/fourseven_scss/.3.4.3.x0c10u++os.linux.x86_64+web.browser+web.cordova/plugin.compileScssBatch.os.linux.x86_64/npm/node_modules/meteor/compileScssBatch/node_modules/node-sass/lib/index.js:16:36)
   at Module._compile (module.js:409:26)
   at Module._extensions..js (module.js:416:10)
   at Object.require.extensions..js (/root/meteor/dev_bundle/lib/node_modules/meteor-babel/register.js:39:5)
   at Module.load (module.js:343:32)
   at Module.Mp.load (/root/meteor/dev_bundle/lib/node_modules/reify/node/runtime.js:16:23)
   at Function.Module._load (module.js:300:12)
   at Module.require (module.js:353:17)
   at require (internal/module.js:12:17)
   at Object.<anonymous> (/tools/isobuild/bundler.js:1570:22)
   at Object.require (/tools/isobuild/bundler.js:1547:11)
   at meteorInstall.node_modules.meteor.compileScssBatch.plugin.compile-scss.js (packages/compileScssBatch/plugin/compile-scss.js:3:18)
   at fileEvaluate (packages/modules-runtime.js:158:9)
   at require (packages/modules-runtime.js:92:16)
   at <runJavaScript-98>:349:1
   at <runJavaScript-98>:355:3

```

**Comment by meteorza** [comment]
Also, running node 4.4.5 with meteor 1.4 branch

**Comment by ramijarrar** [comment]
This issue is also being tracked in meteor/meteor#7297

**Comment by paralin** [comment]
How do you guys go about fixing this?

**Comment by ramijarrar** [comment]
@paralin Take a look at https://github.com/meteor/meteor/issues/7297, at the moment we are either waiting for MDG to provide a new beta release or for @sebakerckhof to update the way node-sass is loaded. You could fork and make the changes yourself if this is urgent for you.

**Comment by sebakerckhof** [comment]
I'll close this, since it's being tracked here: https://github.com/meteor/meteor/issues/7297

**Comment by sebakerckhof** [comment]
Fixed in `1.4-beta.10`, clean your `$HOME/.meteor` and/or `.meteor/local `
